__tests__/__mocks__/job.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: run.googleapis.com/v1
+kind: Job
+metadata:
+  name: my-test-job
+  labels:
+    cloud.googleapis.com/location: ${APP_LOCATION}
+spec:
+  template:
+    metadata:
+      annotations:
+        run.googleapis.com/execution-environment: gen2
+        run.googleapis.com/network-interfaces: '[{"network":"projects/my-test-project/global/networks/my-test-network","subnetwork":"projects/my-test-project/regions/us-east1/subnetworks/us-east1-my-test-network"}]'
+        run.googleapis.com/vpc-access-egress: private-ranges-only
```

Comment:
Maybe better to stay close to default config

Reply:
Good call, cleaned up all unnecessary annotations